src/index/tests/index_tests.rs

```diff
@@ -121,6 +121,70 @@ fn actions_are_indexed() {
     }
 }
 
+#[test]
+fn fb_methods_are_indexed() {
+    let index = index!(
+        r#"
+        FUNCTION_BLOCK myFuncBlock
+            METHOD foo
+                VAR x : SINT; END_VAR
+            END_METHOD
+        END_FUNCTION_BLOCK
+    "#
+    );
+
+    let foo_impl = index.find_implementation("myFuncBlock.foo").unwrap();
+    assert_eq!("myFuncBlock.foo", foo_impl.call_name);
+    assert_eq!("myFuncBlock", foo_impl.type_name);
+    let info = index
+        .get_type("myFuncBlock.foo")
+        .unwrap()
+        .get_type_information();
+    if let crate::typesystem::DataTypeInformation::Struct {
+        name,
+        member_names,
+        varargs: _,
+    } = info
+    {
+        assert_eq!("myFuncBlock.foo_interface", name);
```

Comment:
is this assertion correct? Should the name here be `myFuncBlock.foo_interface` ?

Reply:
Yes, _interface is the correct struct name